src/services/teams/hooks/index.js

```diff
@@ -515,9 +518,32 @@ const testChangesForPermissionRouting = globalHooks.ifNotLocal(async (hook) => {
 			throw new Forbidden('Can not adding users from other schools.');
 		}
 		if (isLeaveTeam) {
-			wait.push(leaveTeam(hook).catch((err) => {
-				throw new Forbidden('Permission LEAVE_TEAM is missing.');
-			}));
+			let amountOfTeamowner = 0; // only counter, do not have to be the real amount of Teamowners!
+			let isTeamowner = false;
+			for (const user of team.userIds) {
```

Comment:
```suggestion
			const users = team.userIds.filter(u=> u.userId.toString() === sessionUserId)
```

Reply:
is not reley more efficent, isn't it? Filter also takes each element in the array, like the for loop, but the for loops still check other things, that have to be checked afterwards... can replace it with an forEach loop